src/plugins/data/common/es_query/kuery/ast/kuery.peg

```diff
@@ -278,6 +278,7 @@ EscapedWhitespace
   = '\\t' { return '\t'; }
   / '\\r' { return '\r'; }
   / '\\n' { return '\n'; }
+  / '\\u00A0' { return '\u00A0'; }
```

Comment:
I think we can replace this with an all-inclusive unicode rule (see https://github.com/pegjs/pegjs/blob/b7b87ea8aeeaa1caf096e2da99fd95a971890ca1/examples/javascript.pegjs#L138)

Reply:
Added. Hope did everything right